src/component/Test.tsx

```diff
@@ -0,0 +1,16 @@
+import Navbar from "./Navbar";
```

Comment:
Is this component used anywhere/can it be removed?

Reply:
I'll remove it. I was using test to experiment with things but it sounds redundant anyway.